doc/ref_model/chapters/chapter03.md

```diff
@@ -171,11 +174,11 @@ Storage resources have the following attributes, with metric definitions that su
 
 <a name="3.2.1.4"></a>
 #### 3.2.1.4 Virtual Network
-This topic is currently covered in [Network](#3.5) section.
+This topic is covered in [Network](#3.5) section.
 
 <a name="3.2.1.5"></a>
 #### 3.2.1.5 Availability Zone
-An availability zone is a logical pool of physical resources (e.g. compute, block storage, and network).  These logical pools segment the physical resources of a cloud based on factors chosen by the cloud operator. The cloud operator may create availability zones based on location (rack, datacenter), or indirect failure domain dependencies like power sources.  Workloads can leverage availability zones to utilise multiple locations or avoid sharing failure domains for a workload, and thus increase its fault-tolerance.
+An availability zone is a logical pool of physical resources (e.g. compute, block storage, and network).  These logical pools segment the physical resources of a cloud based on factors chosen by the cloud operator. The cloud operator may create availability zones based on location (rack, datacenter), or indirect failure domain dependencies like power sources.  Workloads can leverage availability zones to utilise multiple locations or avoid sharing failure domains for a workload, and thus increase the workloads fault-tolerance.
```

Comment:
Shouldn't it be "the workload fault-tolerance"?

Reply:
@walterkozlowski should be: "the workload's fault-tolerance"
Updating